s3/pkg/service/object.go

```diff
@@ -164,6 +164,7 @@ func (s *s3Service) PutObject(ctx context.Context, in pb.S3_PutObjectStream) err
 		limitedDataReader = data
 	}
 
+	actualSize := req.Size
 	// encrypt if needed
 	if bucket.ServerSideEncryption.SseType == "SSE" {
 		byteArr, _ := ioutil.ReadAll(limitedDataReader)
```

Comment:
utils.EncryptWithAES256RandomKey may return error, so need to consider it before continue the next step.

Reply:
added check